### PR TITLE
cracklib 2.9.5

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by nijikon** [comment]
I would really appreciate some feedback on this /cc @xu-cheng @bfontaine @DomT4

**Comment by mikemcquaid** [comment]
This makes sense to me :+1:

**Comment by DomT4** [comment]
Definitely :+1:

File: Library/Formula/cracklib.rb [hunk 1]
```diff
@@ -22,5 +25,17 @@ def install
...
                           "--with-default-dict=#{HOMEBREW_PREFIX}/share/cracklib-words"
     system "make", "install"
+
+    resource("cracklib-words").stage do |r|
+      share.install "cracklib-words-#{r.version}" => "cracklib-words"
+    end
+  end
+
+  def post_install
+    system "#{bin}/cracklib-packer < #{share}/cracklib-words"
```
Comment (bfontaine) [inline Library/Formula/cracklib.rb]: You can probably do `pipe_output("#{bin}/cracklib-packer", (share/"cracklib-words").read, 0)`.

File: Library/Formula/cracklib.rb [hunk 2]
```diff
@@ -22,5 +25,17 @@ def install
...
+  end
+
+  test do
+    assert_match /password: it is based on a dictionary word/, shell_output("echo 'password' | #{bin}/cracklib-check")
```
Comment (bfontaine) [inline Library/Formula/cracklib.rb]: You can use `pipe_output("#{bin}/cracklib-check", "password", 0)`.

File: Library/Formula/cracklib.rb (hunk 1 above)
Comment (dunn) [inline Library/Formula/cracklib.rb]: Out of curiosity, why does that need to be in `post_install`?

Comment (DomT4) [inline Library/Formula/cracklib.rb]: It actually probably doesn't need to be any more, although it is the sort of thing we don't handle directly during install; if nobody else really wants to change it I don't mind where it is now.

Comment (mikemcquaid) [inline Library/Formula/cracklib.rb]: Yeh, putting in install probably makes sense.

Comment (nijikon) [inline Library/Formula/cracklib.rb]: ```pipe_output``` is not available in install section.

Moving code from post_install to install breaks the build.

```
==> Verifying cracklib--cracklib-words-20080507.gz checksum
==> /usr/local/Cellar/cracklib/2.9.5/bin/cracklib-packer < /usr/local/Cellar/cracklib/2.9.5/share/cracklib-words
/usr/local/share/cracklib-words.pwd: Operation not permitted
/usr/local/share/cracklib-words: Operation not permitted
/usr/local/Library/Homebrew/debrew.rb:11:in `raise'
```

Comment (xu-cheng) [inline Library/Formula/cracklib.rb]: I have test it under the sandbox. This should be put in `post_install`.

**Comment by xu-cheng** [comment]
Thanks @nijikon

**Pull request closed without merging**